Containers/onlyoffice/Dockerfile

```diff
@@ -1,4 +1,4 @@
 # From https://github.com/ONLYOFFICE/Docker-DocumentServer/blob/master/Dockerfile
 FROM onlyoffice/documentserver:7.2.2.56
 
-HEALTHCHECK CMD curl -skfI localhost || exit 1
\ No newline at end of file
+HEALTHCHECK CMD nc -z localhost 80 || exit 1
```

Comment:
are you sure that nc is included by default?

Reply:
```
docker run --rm -it --entrypoint nc onlyoffice/documentserver:7.2.2.56
usage: nc [-46CDdFhklNnrStUuvZz] [-I length] [-i interval] [-M ttl]
          [-m minttl] [-O length] [-P proxy_username] [-p source_port]
          [-q seconds] [-s sourceaddr] [-T keyword] [-V rtable] [-W recvlimit]
          [-w timeout] [-X proxy_protocol] [-x proxy_address[:port]]
          [destination] [port]
```

Reply:
okay perfect, thanks! :)